src/components/SpectralProfiler/SpectralProfilerSettingsPanelComponent/SpectralProfilerSettingsPanelComponent.tsx

```diff
@@ -22,6 +22,8 @@ export enum SpectralProfilerSettingsTabs {
 
 @observer
 export class SpectralProfilerSettingsPanelComponent extends React.Component<WidgetProps> {
+    secondaryAxisCursorInfoVisible?: boolean;
```

Comment:
remove unused variable

Reply:
This isn't unused. It is used in 6 places in that file.

Reply:
Those are either for initialization, or in unused functions, or `widgetStore.secondaryAxisCursorInfoVisible`.